src/libData/BlockChainData/BlockLinkChain.h

```diff
@@ -122,6 +122,7 @@ class BlockLinkChain {
   void SetBuiltDSComm(const std::deque<std::pair<PubKey, Peer>>& dsComm) {
     m_builtDsCommittee = dsComm;
   }
+
```

Comment:
the only change in this file seems to be space

Reply:
Yeah, i add a empty line because there should be a empty line between two functions. It is easier to tell these are two functions.